app/src/main/java/com/stevesoltys/seedvault/restore/RestoreActivity.kt

```diff
@@ -19,8 +19,6 @@ class RestoreActivity : RequireProvisioningActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        if (isSetupWizard) hideSystemUI()
```

Comment:
Not needed anymore?

Reply:
I think you said that the system UI should still be shown in SUW, right? The code for doing that changed on Android 11.

Reply:
But I don't see any new code that still does this. Should make a note somewhere if this needs updating.

Reply:
Yes due to #63 I didn't add the new code.

Reply:
Ah, I see the confusion, that shouldn't say *and* - we want it to look like what it currently does / has for the year so far.

Reply:
alright, I added it back.